pages/apis/rest_api/analytics/flaky_tests.md.erb

```diff
@@ -0,0 +1,32 @@
+# Flaky tests API
+
+{:toc}
+
+This feature is currently in beta, and not available to all organizations. If you would like early access, please TBC.
```

Comment:
Let's style this as a warning because it will stop people from making progress if they're not aware they need to request access:

```suggestion
> 🚧 Beta feature
> Flaky test detection is in beta and not available to all organizations. To get early access, TBC.

{:toc}
```

A few nitpicks included in the suggestion:

- Be clear about what "this" refers to.
- Avoid time words like "currently" because they tend to lose meaning when published.
- Use "To X, do Y" structure for the instruction.
- Generally avoid "please" because it weakens the instruction, which is a required action here.

I also switched the order of the table of contents since the warning is important to know before you get into the content of the page.